lib/geo_encode.py

```diff
@@ -0,0 +1,9 @@
+from geopy.geocoders import Nominatim
+
+
+def address_to_geopoint(address):
+
+    geolocator = Nominatim(user_agent="SmartMeet")
```

Comment:
we may have to move to a paying service at some point but if it works, it is perfect. No change yet.

Reply:
Good work, thanks!